### PR TITLE
Add prebuilt drivers configs for ubuntu generic kernel 5.12.3-arch1-1 1

### PR DESCRIPTION
**Pull request by gauravgahlot**
Signed-off-by: Gaurav Gahlot <gauravgahlot0107@gmail.com>

### CODE REVIEW

**Comment by leogr** [comment]
> @leogr: The `/retest` command does not accept any targets.

@poiana , sorry :pray: 

/test build-drivers-ubuntu-generic-5-presubmit

:smile_cat:

**Comment by gauravgahlot** [comment]
@leogr Is there anything I can do to fix the tests?

**Comment by maxgio92** [comment]
Hey @gauravgahlot, we're checking

File: driverkit/config/17f5df52a7d9ed6bb12d3b1768460def8439936d/ubuntu-generic_5.12.3-arch1-1_1.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+kernelversion: 1
+kernelrelease: 5.12.3-arch1-1_1
```
Comment (maxgio92) [inline driverkit/config/17f5df52a7d9ed6bb12d3b1768460def8439936d/ubuntu-generic_5.12.3-arch1-1_1.yaml:2]:
```suggestion
kernelrelease: 5.12.3-arch1-1
```

File: driverkit/config/2aa88dcf6243982697811df4c1b484bcbe9488a2/ubuntu-generic_5.12.3-arch1-1_1.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+kernelversion: 1
+kernelrelease: 5.12.3-arch1-1_1
```
Comment (maxgio92) [inline driverkit/config/2aa88dcf6243982697811df4c1b484bcbe9488a2/ubuntu-generic_5.12.3-arch1-1_1.yaml:2]:
```suggestion
kernelrelease: 5.12.3-arch1-1
```

File: driverkit/config/5c0b863ddade7a45568c0ac97d037422c9efb750/ubuntu-generic_5.12.3-arch1-1_1.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+kernelversion: 1
+kernelrelease: 5.12.3-arch1-1_1
```
Comment (maxgio92) [inline driverkit/config/5c0b863ddade7a45568c0ac97d037422c9efb750/ubuntu-generic_5.12.3-arch1-1_1.yaml:2]:
```suggestion
kernelrelease: 5.12.3-arch1-1
```

File: driverkit/config/85c88952b018fdbce2464222c3303229f5bfcfad/ubuntu-generic_5.12.3-arch1-1_1.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+kernelversion: 1
+kernelrelease: 5.12.3-arch1-1_1
```
Comment (maxgio92) [inline driverkit/config/85c88952b018fdbce2464222c3303229f5bfcfad/ubuntu-generic_5.12.3-arch1-1_1.yaml:2]:
```suggestion
kernelrelease: 5.12.3-arch1-1
```

File: driverkit/config/ae104eb20ff0198a5dcb0c91cc36c86e7c3f25c7/ubuntu-generic_5.12.3-arch1-1_1.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+kernelversion: 1
+kernelrelease: 5.12.3-arch1-1_1
```
Comment (maxgio92) [inline driverkit/config/ae104eb20ff0198a5dcb0c91cc36c86e7c3f25c7/ubuntu-generic_5.12.3-arch1-1_1.yaml:2]:
```suggestion
kernelrelease: 5.12.3-arch1-1
```

**Review by maxgio92 — CHANGES_REQUESTED** [review]
Hey @gauravgahlot, thank you. I left some suggestions, as the `kernelrelease` in the Driverkit config should contain the kernel release only, without the version (as for `uname -r`) :)

**Comment by gauravgahlot** [comment]
Thank you @maxgio92. I have made the suggested changes. :rocket:

**Review by maxgio92 — APPROVED** [review]
/approve

Thank you @gauravgahlot! :)

**Pull request merged**